synapse/federation/transport/server.py

```diff
@@ -550,24 +550,24 @@ async def on_GET(self, origin, content, query, context, event_id):
 
 
 class FederationV1SendJoinServlet(BaseFederationServlet):
-    PATH = "/send_join/(?P<context>[^/]*)/(?P<event_id>[^/]*)"
+    PATH = "/send_join/(?P<room_id>[^/]*)/(?P<event_id>[^/]*)"
```

Comment:
Should we also update `Federation*InviteServlet` to have these be `room_id` instead of `context`? It doesn't seem they have the same bug, but consistency is nice!

Reply:
happy to do so if you like.

Reply:
I think we should do it while we're in here! Thanks!

Reply:
well if we're going that far, let's get rid of them all. done.